go.mod

```diff
@@ -5,6 +5,8 @@ go 1.14
 require (
 	github.com/antonmedv/expr v1.8.2
 	github.com/cenkalti/backoff/v4 v4.0.2
+	github.com/golangci/golangci-lint v1.37.1 // indirect
+	github.com/google/addlicense v0.0.0-20200906110928-a0294312aa76 // indirect
```

Comment:
I don't think we need to include these in the module, and believe we should try not to.

Neither the collector or collector-contrib repos have needed these in `go.mod`.